src/batou/secrets/edit.py

```diff
@@ -57,7 +54,7 @@ def interact(self):
                 print("Your changes are still available. You can try:")
                 print("\tedit       -- opens editor with current data again")
                 print("\tencrypt    -- tries to encrypt current data again")
-                print("\tquit       -- quits and loses your changes")
+                print("\tquit       -- aborts editing")
```

Comment:
can we leave the indicator about loosing changes in there?

Reply:
It would contradict itself with deleting the file in case no recipients are entered. Semantically it would probably be better to have the option to delete the secrets file here.

If you want to keep the old wording, I can revert that.

Reply:
Alright, as discussed we need to cater to the case where someone created a secrets file, added members, hasn't committed it yet and then edits it again but fat-fingers the member list. In that case if we determine that 'delete the file' is an option this needs to become an additional option to select and quit only looses the changes you made since last saving.

Reply:
As the changes would be larger, I created a new issue for that https://github.com/flyingcircusio/batou/issues/234 and restored the old wording for now. This PR still solves the initial issue of creating zero bytes secrets files, which cannot be edited afterwards. I would vote for a merge here.